kevm_pyk/solc_to_k.py

```diff
@@ -229,10 +238,10 @@ def extract_rule(name, inputs):
     def extract_lhs(name, input_names):
         # TODO: add structure to LHS:
         # generate (uncurried) unparser, function name, and list of arguments
-        return KToken(f'{contract_name}.{name}(' + ', '.join(input_names) + ')', None)
+        return KToken(f'{contract_name}.{name}(' + ', '.join(input_names) + ')', 'ByteArray')
```

Comment:
:question: Should these sort names not be wrapped by `KSort`? (Not that it matters much, since `KToken` is used here as replacement for a propert `KAst`.)

```suggestion
        return KToken(f'{contract_name}.{name}(' + ', '.join(input_names) + ')', KSort('ByteArray'))
```

Reply:
For example here: https://github.com/runtimeverification/k/blob/master/k-distribution/src/main/scripts/lib/pyk/kast.py#L214